app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -68,45 +70,71 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
+            var xhr = new XMLHttpRequest()
+            var url = $form.getAttribute('action')
+            var params = new FormData($form)
+            params = new URLSearchParams(params).toString()
+
+            xhr.open('POST', url, true)
+            xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded')
+
+            xhr.onreadystatechange = function () {
+              if (xhr.readyState === 4 && xhr.status === 200) {
+                trackEvent(getTrackEventParams($form))
+                showFormSuccess(xhr.message)
+                revealInitialPrompt()
+                setInitialAriaAttributes()
+                that.activeForm.classList.toggle(jshiddenClass)
+              } else {
+                showError(xhr)
+                enableSubmitFormButton($form)
+              }
+            }
+
+            disableSubmitFormButton($form)
+            xhr.send(params)
+          })
+        }
+      }
 
       function disableSubmitFormButton ($form) {
-        $form.find('input[type="submit"]').prop('disabled', true)
+        var formButton = $form.querySelector('[type="submit"]')
+        formButton.setAttribute('disabled', true)
       }
 
       function enableSubmitFormButton ($form) {
-        $form.find('input[type="submit"]').removeAttr('disabled')
+        var formButton = $form.querySelector('[type="submit"]')
+        formButton.removeAttribute('disabled')
       }
 
       function setInitialAriaAttributes () {
-        that.$forms.attr('aria-hidden', true)
-        that.$pageIsNotUsefulButton.attr('aria-expanded', false)
-        that.$somethingIsWrongButton.attr('aria-expanded', false)
+        for (var i = 0; i < that.forms.length; i++) {
+          that.forms[i].setAttribute('aria-hidden', true)
+        }
+        that.pageIsNotUsefulButton.setAttribute('aria-expanded', false)
+        that.somethingIsWrongButton.setAttribute('aria-expanded', false)
       }
 
       function setHiddenValues () {
-        that.$somethingIsWrongForm.append('<input type="hidden" name="javascript_enabled" value="true"/>')
-        that.$somethingIsWrongForm.append($('<input type="hidden" name="referrer">').val(document.referrer || 'unknown'))
+        var javascriptEnabled = document.createElement('input')
+        javascriptEnabled.setAttribute('type', 'hidden')
+        javascriptEnabled.setAttribute('name', 'javascript_enabled')
+        javascriptEnabled.setAttribute('value', true)
+        that.somethingIsWrongForm.appendChild(javascriptEnabled)
+
+        var referrer = document.createElement('input')
+        referrer.setAttribute('type', 'hidden')
+        referrer.setAttribute('name', 'referrer')
+        referrer.setAttribute('value', document.referrer || 'unknown')
+        that.somethingIsWrongForm.appendChild(referrer)
         that.somethingIsWrongForm.invalidInfoError = [
           '<h2>',
           '  Sorry, we’re unable to send your message as you haven’t given us any information.',
```

Comment:
Could the double space be removed?

```suggestion
          'Sorry, we’re unable to send your message as you haven’t given us any information.',
```

Reply:
Good point, have tidied up a bit.